### PR TITLE
Remove logging_agent_in_visibility feature-toggle

### PR DESCRIPTION
**Pull request by astarnell**
Signed-off-by: Alexander Starnell <alexander.starnell@zalando.de>

### CODE REVIEW

File: cluster/manifests/deletions.yaml [hunk 1]
```diff
@@ -1,9 +1,6 @@
 # everything defined under here will be deleted before applying the manifests
-pre_apply:
+pre_apply: []
 {{ if eq .LifecycleStatus "ready" }}
```
Comment (aermakov-zalando) [inline cluster/manifests/deletions.yaml]: Please drop the whole block, it's not needed anymore.

Comment (astarnell) [reply cluster/manifests/deletions.yaml]: done

**Comment by aermakov-zalando** [comment]
👍

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**